### ISSUE TITLE
moin.wiki site

### ISSUE THREAD

**Issue by ThomasWaldmann** [moinwiki/moin#1666]
Quite a while ago, I registered the `moin.wiki` domain (nothing to see there yet).

Considering moin2 is very different from moin 1.x and also that there is quite a lot of outdated and rather not interesting content (from a moin2 perspective) on the `moinmo.in` site, we could have a clean start on the new domain.

Not sure about **when** to do that, so please give feedback.

**Comment by UlrichB22** [comment]
I would appreciate a new moin2 wiki, it would definitely be a good reference system.

**Comment by RogerHaase** [comment]
Agree, the moin2 info is hard to find on moinmo.in.  Start moin.wiki now, add a bit of info, announce in a month from now.
